server/services/participants.js

```diff
@@ -466,8 +466,8 @@ const makeParticipant = async (participantJson) => {
   return res;
 };
 
-const createParticipantUserMap = async (userId, email) => {
-  const participants = await dbClient.db[collections.PARTICIPANTS]
+const createParticipantUserMap = async (userId, email, tnx) => {
```

Comment:
this is a bit of a nit, but we might want a more descriptive variable name than tnx. I have no idea what it stands for.

Reply:
transaction, sorry for this, I will update this in my next PR, I'm currently working on same pice of code